### PR TITLE
upgrade to pdex-plan-net 0.1.5

### PR DESCRIPTION
**Pull request by lmsurpre**

### CODE REVIEW

**Comment by prb112** [comment]
```
 .................................................
Error:  Tests run: 4, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 8.651 s <<< FAILURE! - in TestSuite
Error:  testGetResources(com.ibm.fhir.ig.davinci.plan.net.test.PlanNetResourceProviderTest)  Time elapsed: 0.001 s  <<< FAILURE!
java.lang.AssertionError: expected [106] but found [110]
	at com.ibm.fhir.ig.davinci.plan.net.test.PlanNetResourceProviderTest.testGetResources(PlanNetResourceProviderTest.java:19)

[INFO] 
[INFO] Results:
[INFO] 
Error:  Failures: 
Error:    PlanNetResourceProviderTest.testGetResources:19 expected [106] but found [110]
[INFO] 
Error:  Tests run: 4, Failures: 1, Errors: 0, Skipped: 0
```

**Review by prb112 — APPROVED** [review]
LGTM - please fix the test :)

File: conformance/fhir-ig-davinci-pdex-plan-net/src/main/resources/hl7/fhir/us/davinci-pdex-plan-net/package/.index.json [hunk 1]
```diff
@@ -6,21 +6,21 @@
       "resourceType": "ImplementationGuide",
       "id": "hl7.fhir.us.davinci-pdex-plan-net",
       "url": "http://hl7.org/fhir/us/davinci-pdex-plan-net/ImplementationGuide/hl7.fhir.us.davinci-pdex-plan-net",
-      "version": "0.3.0"
+      "version": "0.1.5"
```
Comment (prb112) [inline conformance/fhir-ig-davinci-pdex-plan-net/src/main/resources/hl7/fhir/us/davinci-pdex-plan-net/package/.index.json:9]: Should we add a release note warning?

Comment (prb112) [reply conformance/fhir-ig-davinci-pdex-plan-net/src/main/resources/hl7/fhir/us/davinci-pdex-plan-net/package/.index.json:9]: more specifically to the version change

Comment (lmsurpre) [reply conformance/fhir-ig-davinci-pdex-plan-net/src/main/resources/hl7/fhir/us/davinci-pdex-plan-net/package/.index.json:9]: sure

**Comment by lmsurpre** [comment]
> please fix the test
fixed

File: conformance/fhir-ig-davinci-pdex-plan-net/src/main/resources/hl7/fhir/us/davinci-pdex-plan-net/package/.index.json (hunk 1 above)
Comment (lmsurpre) [reply conformance/fhir-ig-davinci-pdex-plan-net/src/main/resources/hl7/fhir/us/davinci-pdex-plan-net/package/.index.json:9]: added the following in the draft release notes:
> Note that the previous packaged version of this IG was version `0.3.0`, but was subsequently "fixed" to indicate that this is still a revision of STU1 Ballot 1 and hence has been dropped to `0.1.5`

**Pull request merged**